README.md

```diff
@@ -30,12 +30,23 @@ Requirements:
 
 - go 1.6 or higher
 - A [working golang](https://golang.org/doc/code.html) environment
+- A [properly configured](https://golang.org/doc/code.html#Organization) Go workspace
 
-From the project root directory, run:
+In your terminal, run:
 ```
+cd $GOPATH
```

Comment:
`$GOPATH` can container multiple paths, so these instructions don't really work in all cases.

We're going to link out to the main Go setup documentation, where it can be properly maintained, to avoid bitrot here.